.github/pull_request_template.md

```diff
@@ -7,21 +7,22 @@
 ## What should reviewers focus on?
 
 
-
 ## Type of Change
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Code refactor (non-breaking change that doesn't add functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] This change requires a documentation update
+- [ ] Bugfix
+- [ ] New feature
+- [ ] Non-breaking change (a change that doesn't alter functionality - i.e., code refactor, configs, etc.)
+
+## Checklist
```

Comment:
Thanks for making the changes! Only thing I'd say is to move this section below the additional release notes section, so that it doesn't break the flow (non-breaking change and breaking change). Otherwise it's a 🚢 for me.